src/components/calcite-tooltip/calcite-tooltip.scss

```diff
@@ -2,16 +2,16 @@
 @include popperArrow();
 
 .container {
-  position: relative;
-  background: var(--calcite-ui-foreground-1);
-  max-width: 300px;
-  max-height: 300px;
-  display: flex;
-  justify-content: flex-start;
-  flex-direction: column;
-  font-weight: 500;
-  color: var(--calcite-ui-text-1);
-  padding: 12px 16px;
-  overflow: hidden;
-  @include font-size(-3);
+  @apply relative 
+    bg-foreground-1 
+    flex 
+    justify-start 
+    flex-col 
+    font-medium 
+    py-3 
+    px-4 
+    overflow-hidden 
+    text--3-wrap;
+  max-width: 18.75rem;
+  max-height: 18.75rem;
```

Comment:
bonus points if we can snap this to an even whole number e.g. 18 or 20

Reply:
What would I use? spacing only goes up to 16rem

Reply:
It looks like extrapolating would give us something like `max-width: 20rem`. This would be ~`w-72`. Not suggesting we modify the config here - just hard code it.

Reply:
I hard codded to 20rem, this is what you mean right?